### PR TITLE
Guts Computer3

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Labeled as a refactor because this doesn't impact players and TG's modular computers has completely replaced this.

No CL as this is purely code cleanup.

### CODE REVIEW

**Comment by alexkar598** [comment]
Don't laptops use computer3?

**Comment by SamHPurp** [comment]
@alexkar598 the old ones do. They now use modular_computer instead

File: code/modules/computer3/NTOS.dm [hunk 1]
```diff
@@ -1,239 +0,0 @@
...
-/*
-	Generate a basic list of files in the selected scope
-*/
-
-/datum/file/program/ntos/proc/list_files()
-	if(!computer || !current) return null
-	return current.files
-
-
-/datum/file/program/ntos/proc/filegrid(var/list/filelist)
```
Comment (Crazylemon64) [inline code/modules/computer3/NTOS.dm]: Hmm, this UI component is one of the things I'd like to bring over to the modular computers system

**Comment by Crazylemon64** [comment]
Eh whatever rip

**Pull request merged**